src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -224,11 +234,20 @@ async def Generate(
                 prompt=req.text,
                 prompt_token_ids=input_ids,
             )
+            if _vllm_tracing_available:
+                is_tracing_enabled = await self.engine.is_tracing_enabled()
+                headers = dict(context.invocation_metadata())
+                trace_headers = None
+                if is_tracing_enabled:
+                    trace_headers = extract_trace_headers(headers)
+                elif contains_trace_headers(headers):
+                    log_tracing_disabled_warning()
```

Comment:
missing the kwargs definition here

Reply:
oh whoops, GH review comment missed that one